string-constants-lib/string-constants/private/german-string-constants.rkt

```diff
@@ -180,7 +180,7 @@
  (online-expansion-error-margin "am Rand")
  ; the label of a preference in the (string-constant online-expansion) section
  (show-arrows-on-mouseover "Bindungen und Tail-Positionen unter Mauszeiger anzeigen")
- (show-blueboxes "Blaue Kästen  und blaue anzeigen")
+ (show-blueboxes "Blaue Kästen und blauen Viertelkreis anzeigen")
```

Comment:
This translation is based on the english one. However, I think it is not actually very helpful. Maybe something similar to "Show blue documentation boxes" (Blauen Hilfe-Kasten anzeigen) would be better?